code/modules/mob/living/carbon/human/death.dm

```diff
@@ -12,11 +12,11 @@
 	if(buckled)
 		buckled.unbuckle_mob(src, force = TRUE)
 
-	Stun(100, TRUE, TRUE)//hold them still as they get deleted so they don't fuck up the animation
 	notransform = TRUE
+
+	QDEL_IN(src, 2 SECONDS) // since this is sometimes called in
 	dust_animation()
 	spawn_dust(just_ash)
-	QDEL_IN(src, 20) // since this is sometimes called in
```

Comment:
qdel in should still be the last thing to go

Reply:
Well then i have no clue what might be breaking it and theres no reason for this PR